src/test/cli/rbd/help.t

```diff
@@ -672,22 +682,28 @@
   rbd help device unmap
   usage: rbd device unmap [--device-type <device-type>] [--pool <pool>] 
                           [--namespace <namespace>] [--image <image>] 
-                          [--snap <snap>] [--options <options>] 
-                          <image-or-snap-or-device-spec> 
+                          [--snap <snap>] [--snap-id <snap-id>] 
+                          [--options <options>] 
+                          <image-or-snap-or-device-spec> <snap-spec> 
```

Comment:
Same as for `rbd device detach` -- `<image-or-snap-or-device-spec>` should remain to be a single positional argument.